keras_nlp/models/distil_bert/distil_bert_presets.py

```diff
@@ -24,6 +24,7 @@
             "params": 66362880,
             "official_name": "DistilBERT",
             "path": "distil_bert",
+            "model_card": "https://huggingface.co/distilbert-base-uncased",
```

Comment:
Instead of linking this, let's link the model card explicitly, So, in this case, https://huggingface.co/distilbert-base-uncased#distilbert-base-model-uncased. Same for other DistilBERT presets.

Reply:
I'm ok with the main link actually. What is here is what you get when you click the model card tab on huggingface. (and i would somewhat suspect that anchor link might change down the link anyway)